src/pug/blog/recap_year_2.pug

```diff
@@ -0,0 +1,97 @@
+      li Option to #[em load / save / copy / reset] profile slots.
+      li Option for #[em longer gyro calibration].
+      li Option to manually #[em tweak gyro calibration] values.
+      li Option to #[em swap gyros] (left <-> right).
+      li Option to #[em invert touch sensor ]polarity.
+      li Ctrl app logs #[em auto-diagnose] and report about common issues.
+      li Ctrl app logs option to enable #[em additional logs]. (eg, touch sensor specific logs).
+
+  h4.bullet Migration from Blender to Build123d
+  p We finally decided on a better CAD tool to replace Blender, that tool is #[+ax('https://build123d.readthedocs.io') build123d], a Python library to declare #[em CAD as code] and uses #[em OpenCASCADE] geometry engine internally (the same than #[em FreeCAD] uses).
+  p As for today we already ported:
+  ul: ul
+    li #[em DPad], #[em Select] and #[em ABXY] buttons.
+    li #[em L1/R1] triggers.
+    li #[em Scrollwheel] assemble.
+    li #[em Battery cover].
+  p With the release of Alpakka 1.0 we also expect to port or create:
+  ul: ul
+    li Main #[em Thumbstick].
+    li New secondary #[em Mini-thumbstick].
+    li New #[em Scrollwheel] assemble.
+    li New #[em Dongle case].
+
+  h4.bullet Alpakka kits in the shop
+  p During summer we introducing the concept of #[em Complete DIY kits] in the shop, while these look like an additional product, they are configurators that help users adding all the relevant parts into the cart, depending on how they want to approach their DIY project. This reduced the guesswork, and was well received by the community.
+
+  h4.bullet Reaching 1000+ members in Discord
+  p Input Labs' most awesome success is our community, always willing to help each other, sharing mods, cool designs, profiles, knowledge and fun. Keep being wonderful, folks 🤍.
+
+  h4.bullet Patents
+  p Since we want to be fully transparent with the community, we explained #[+a('patents') our strategy] regarding patents. In a nutshell, we submitted 2 of our innovations, in both European and US patent offices.
+
+  h4.bullet Many wireless prototype iterations
+  p We created a total of:
+  ul
+    li #[em 10 RF module prototypes] (later renamed as Marmota module).
+    li #[em 2 Alpakka carrier board prototypes] compatible with such RF modules.
+    li #[em 4 breakout boards] to perform hardware integration tests.
+
+  h4.bullet Alpakka 1.0 announcement and pre-order
+  p All these efforts on the wireless development eventually crystallized into the the #[+a('alpakka_v1') Alpakka 1.0 reference design], sadly we were not able to have it ready for the holiday season, since we are still putting the finishing touches.
+
+  hr
+  h3 Things that could have gone better
+
+  p There is an elephant in the room here: The development of the wireless Alpakka, in which we hit many research dead-ends, and it consumed most of our time during the whole year.
```

Comment:
Well, I wouldn't call it the elephant in the room. It was legitimate learning, and we received a lot of tips and feedback from the community. Hindsight is always 20/20, and we did our part. I think it's about par for the course.
So, I would rephrase this section.